docs/sources/service.md

```diff
@@ -0,0 +1,108 @@
+# Ingress source
+
+The service source creates DNS entries based on `Service` resources.
+
+## Filtering the Services considered
+
+The `--service-type-filter` flag filters Service resources by their `spec.type`.
+The flag may be specified multiple times to allow multiple service types.
+
+This source supports the `--label-filter` flag, which filters Service resources
+by a set of labels.
+
+## Domain names
+
+The domain names of the DNS entries created from a Service are sourced from the following places:
```

Comment:
```suggestion
The domain names of the DNS entries created from a Service are sourced following this plan:
```

Reply:
I don't think "plan" is quite the right word. It is an algorithm, not a proposal. It does specify the locations where the data are sourced, so "places" isn't too far off, though the items do describe the act of adding instead of the data being added.